pako/pako_manager.py

```diff
@@ -91,6 +91,26 @@ def install_one(self, package: str, fmt: str = None) -> bool:
             if self.call(self.config['install'].split() + [name.format(package)]) == 0:
                 return True
         return False
+    
+    def uninstall_one(self, package: str, fmt: str = None) -> bool:
```

Comment:
With this being almost identical to `install_one` (and same for install vs uninstall), I wonder if we refactor it a little to call a common function.

Maybe something like:

```Python
def install_one(self, package: str, fmt: str = None) -> bool:
    return self.action_one('uninstall', **kwargs)

def uninstall_one(self, package: str, fmt: str = None) -> bool:
    return self.action_one('uninstall', **kwargs)

def action_one(self, action: str, package: str, fmt: str = None) -> bool:
    # existing code
```

This way we retain the current API but reduce the overall code. This should make future changes quicker and easier. It's not always obvious when two methods are the same, you have to read them very carefully to check, there's more chance of someone change one and not the other, or diverging them unintentionally.

Reply:
I have improve the Code